sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1868,9 +1917,13 @@ export interface SearchResourceEncryptionKey {
 // @public
 export type SearchResult<T> = {
     readonly score: number;
+    readonly rerankerScore?: number;
     readonly highlights?: {
         [k in keyof T]?: string[];
     };
+    readonly captions?: {
+        [propertyName: string]: CaptionResult[];
+    } | null;
```

Comment:
will this `| null` go away as well?

Reply:
Yes. This null will go away when this issue is resolved. (By next release in 2 weeks)